packages/editor/src/components/parts/process-data/ProcessDataPart.tsx

```diff
@@ -0,0 +1,45 @@
+import { PartProps, usePartDirty, usePartState } from '../../editors';
+import { Input, MessageText, useFieldset } from '../../widgets';
+import { PathContext, useValidations } from '../../../context';
+import { useProcessDataData } from './useProcessDataData';
+import { IVY_SCRIPT_TYPES, ProcessDataData } from '@axonivy/inscription-protocol';
+import { PathFieldset } from '../common';
+
+export function useProcessDataPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = useProcessDataData();
+  const compareData = (data: ProcessDataData) => [data.data];
+  const validations = useValidations(['data']);
+  const state = usePartState(compareData(defaultConfig), compareData(config), validations);
+  const dirty = usePartDirty(compareData(initConfig), compareData(config));
+  return {
+    name: 'Process Data',
+    state,
+    reset: { dirty, action: () => reset() },
+    content: <ProcessDataPart />
+  };
+}
+
+const ProcessDataPart = () => {
+  const { config, update } = useProcessDataData();
+
+  const dataClassField = useFieldset();
+
+  return (
+    <PathContext path='data'>
+      <PathFieldset label='Data Class' {...dataClassField.labelProps} path='data'>

```

Comment:
remove this path context. or you will endup with a path context "data.data" which is wrong. The PathFieldset is just fine.
If you have the error that you can not put to components in the same return level you can simply use a jsx fragmet around it 
```js
<>
  <PathFieldset ... />
  <MessageText ... />
</>
```